BedrockCommand.cpp

```diff
@@ -104,22 +110,81 @@ void BedrockCommand::_init() {
     }
 }
 
+void BedrockCommand::startTiming(TIMING_INFO type) {
+    if (get<0>(_inProgressTiming) != INVALID ||
```

Comment:
The integer `<N>` is a template parameter to `std::get`. I agree it's an awkward syntax, but it's the standard for `get`. It'd be easy to ask "Why not do `_inProgressTiming.get(N)`?" It looks nicer, but the members of a tuple can be of different types, and the templated version lets the compiler choose the function with the correct return type for each index.

Imagine a tuple `tuple<int, string>`.

If we wanted `tuple::get(index)`, we wouldn't be able to specify the return type to be different for `0` and `1`.

Using the template argument for the index let's the compiler select the function with the correct return type, but it does look kinda strange. At least, being in the standard library, there are lots of examples on the internet.